app/views/layouts/adopter_foster_dashboard.html.erb

```diff
@@ -1,118 +1,117 @@
-                          <% end %>
-                        </li>
-                      </ul>
-                      <!-- Navbar header -->
-                      <span class="navbar-header">Account Settings</span>
-                      <ul class="list-unstyled ms-n2 mb-0">
-                        <!-- Nav item -->
-                        <li class="nav-item">
-                          <%= active_link_to edit_user_registration_path, class: "nav-link" do %>
-                            <i class="fe fe-settings nav-icon"></i> Edit Settings
-                          <% end %>
-                        </li>
-                        <!-- Nav item -->
-                        <li class="nav-item">
-                          <%= button_to destroy_user_session_path, method: :delete, data: { turbo: false }, class: 'nav-link' do %>
-                            <i class="fe fe-power nav-icon"></i> Log Out
-                          <% end %>
-                        </li>
-                        <li class="nav-item">
-                          <%= active_link_to adopter_fosterer_dashboard_index_path, class: "nav-link" do %>
-                            <i class="fe fe-trash nav-icon"></i> Delete Account
-                          <% end %>
-                        </li>
-                      </ul>
-                    </div>
+          <div class="row mt-0 mt-md-4">
+            <div class="col-lg-3 col-md-4 col-12">
+              <!-- Side navbar -->
+              <nav class="navbar navbar-expand-md shadow-sm mb-4 mb-lg-0 sidenav">
+                <!-- Menu -->
+                <a class="d-xl-none d-lg-none d-md-none text-inherit fw-bold" href="#">Menu</a>
+                <!-- Button -->
+                <button class="navbar-toggler d-md-none icon-shape icon-sm rounded bg-primary text-light" type="button" data-bs-toggle="collapse" data-bs-target="#sidenav" aria-controls="sidenav" aria-expanded="false" aria-label="Toggle navigation">
+                  <span class="fe fe-menu"></span>
+                </button>
+                <!-- Collapse navbar -->
+                <div class="collapse navbar-collapse" id="sidenav">
+                  <div class="navbar-nav flex-column">
+                    <span class="navbar-header">Dashboard</span>
+                    <ul class="list-unstyled ms-n2 mb-4">
+                      <!-- Nav item -->
+                      <li class="nav-item">
+                        <%= active_link_to adopter_fosterer_dashboard_index_path, class: "nav-link" do %>
+                          <i class="fe fe-home nav-icon"></i> My Dashboard
+                        <% end %>
+                      </li>
+                      <!-- Nav item -->
+                      <li class="nav-item">
+                        <%= active_link_to adopter_fosterer_adopter_applications_path, class: "nav-link" do %>
+                          <i class="fe fe-book nav-icon"></i> <%= t("dashboard.applications.header_title")%>
```

Comment:
Sorry the diff is showing everything. This is the only real change for this file.